ONSdigital/eq-terraform
# Remove submitter from terraform

**What**
Inline with revised downstream integration with data exchange team,
we remove the submitter from our architecture and consider the Rabbitmq
queue to be our handoff with the data exchange team.

**How to test**
1. Checkout the branch and run the terraform job.
2. Check that the submitter machines haven't been created.
3. Check that the deploy_submitter bash script hasn't tried to be run.
